build-tools/packages/version-tools/src/versions.ts

```diff
@@ -33,7 +34,20 @@ export function getSimpleVersion(
 		}
 
 		if (!argRelease) {
-			version = changePreReleaseIdentifier(version, "dev");
+			let preReleaseIdentifier = "dev";
+
+			switch (true) {
+				case packageTypesField === "beta":
+					preReleaseIdentifier = "dev-beta-types";
```

Comment:
should there be version bag for public releases? (similar to alpha as `dev-alpha-types`) I'm confused between public and untrimmed versioning.